go2ts.go

```diff
@@ -13,15 +13,21 @@ import (
 
 // Go2TS writes TypeScript interface definitions for the given Go structs.
 type Go2TS struct {
-	structs        []structRep
-	seen           map[reflect.Type]structRep
+	interfaces []interfaceDefinition
+	types      []typeDefinition
+
+	// seen maps from a reflect.Type that's already been seen to its TypeScript
+	// type name.
+	seen map[reflect.Type]string
+
+	// anonymousCount keeps track of the number of anonymous structs we've had to name.
```

Comment:
s/structs/types/

Reply:
Only anyonymous structs get names, not any other kinds, so keeping the comment the same.

Reply:
Only done for structs, keeping the comment as is.